kivy/uix/scrollview.py

```diff
@@ -276,6 +276,21 @@ def _set_do_scroll(self, value):
     (:attr:`do_scroll_x` + :attr:`do_scroll_y`)
     '''
 
+    always_overscroll = BooleanProperty(True)
+    '''Make sure user can overscroll even if there is not enough content
+    to require scrolling.
+
+    This is useful if you want to trigger some action on overscroll, but
+    there is not always enough content to trigger it.
+
+    :attr:`always_overscroll` is a
+    :class:`~kivy.properties.BooleanProperty` and defaults to `True`.
+    versionadded:: 2.0.0
+    The option was added and enabled by default, set to False to get the
```

Comment:
@tshirtman Shouldn't there be a new line before `versionadded` tag?

Reply:
hm, maybe, i'll add it along the unittest fix, good call.

Reply:
Maybe docs won't be generated property without new line.
Also, better readability :)

Reply:
yes, and the `..` before was missing as well, good thing you mentioned it.